src/Hangfire.Core/Dashboard/Pager.cs

```diff
@@ -26,42 +26,57 @@ public class Pager
 
         private int _startPageIndex = 1;
         private int _endPageIndex = 1;
-
-        public Pager(int from, int perPage, long total)
+                
+        public Pager(int from, int perPage, long total, string filterString = null, string startDate = null, string endDate = null)
         {
+            JobsFilterText = string.IsNullOrEmpty(filterString) ? null : filterString;
```

Comment:
get rid of filterString, startDate and endDate from constructor and let it be set through properties don't like the use of a lot of default params